Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: install install-dev tests style docs docs-serve
+
+install:
+	pip install sqlglot
+
+install-dev:
+	pip install -e ".[dev]"
+	pre-commit install
```

Comment:
we should not install pre-commit

Reply:
Should we include a separate rule for installing it, or just ignore its installation completely?

Reply:
To expand on this: `pre-commit install` actually installs the git hooks for pre-commit which Toby wants to keep optional: https://pre-commit.com/#3-install-the-git-hook-scripts

So just having pre-commit pip-install allows the style command to work for everyone but then having a separate `install-pre-commit:` lets someone opt-in to having the commit hooks run automatically. 
```
install-pre-commit:
	pre-commit install
```

Reply:
Done.